src/audio/lib/data.c

```diff
@@ -1,5 +1,200 @@
 #include "global.h"
 
+s16 D_801D1E80[][128] ALIGNED(16) = {
```

Comment:
Thanks for this. Could you add a small comment here saying it's unused but inferred to be used in audio synthesis based off animal crossing audio?